packages/commerce/backend/src/config/env.ts

```diff
@@ -3,9 +3,18 @@ import { z } from 'zod'
 const envSchema = z.object({
   PORT: z.coerce.number().default(3004),
   NODE_ENV: z.string().default('development'),
+  FRONTEND_URL: z.string().default('http://localhost:4004'),
   DATABASE_URL: z
     .string()
-    .default('postgres://postgres:password@localhost:5433/commerce_backend')
+    .default('postgres://postgres:password@localhost:5433/commerce_backend'),
+  AUTH_SERVER: z.string().url().default('http://rafiki-auth:3006'),
+  PAYMENT_POINTER: z.string().default('http://rafiki-backend/shop'),
+  KEY_ID: z.string().default('8e5e109e-9ea8-44a0-9490-1792bf1e07cb'),
+  PRIVATE_KEY: z
+    .string()
+    .default(
+      'LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tCk1DNENBUUF3QlFZREsyVndCQ0lFSU9LemxweFR5c0xhbS94VWZ5R2t2T2dpSmcraTEvT25STURWMElqNi9vVnQKLS0tLS1FTkQgUFJJVkFURSBLRVktLS0tLQ=='
+    )
```

Comment:
We will need to seed our wallet with a default account with two payment pointers. Will do this in a separate PR later on.

Reply:
why 2 payment pointers?

Reply:
One that will be the shop's payment pointer and one that will act as the customer.

Reply:
I don't think the customer one is necessary as any payment pointer can be used for customer actions